content/en/docs/Tracing/tempo-walkthrough.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Grafana Tempo tracing
+description: Tracing with Grafana Tempo.
```

Comment:
Maybe something like, "A short proof-of-concept tutorial of tracing using Grafana Tempo."